mongo/database.go

```diff
@@ -180,6 +180,9 @@ func (db *Database) processRunCommand(ctx context.Context, cmd interface{},
 // If the command document contains a session ID or any transaction-specific fields, the behavior is undefined.
 //
 // The opts parameter can be used to specify options for this operation (see the options.RunCmdOptions documentation).
+//
+// Specifying API versioning options in the command document and declaring an API version on the client is not supported.
```

Comment:
I'd move this up to the second paragraph as it already discusses a similar case for session/transaction fields.

Reply:
Moved to the end of the second paragraph.